README.md

```diff
@@ -1,2 +1,71 @@
-# Hoverpod
-Srishti 2019
+## Hoverpod
+_A vehicle that hovers over land as well as water_
+![bot1](https://github.com/shubhamverma989630/Hoverpod/blob/master/IMG_20190308_003335_Bokeh.jpg)
+## Abstract
+The idea of this project is to make a vehicle that hovers over water and land.The team is working on this project for three months and team was successful in making a hoverpod that can lift upto 15 kgs.Moreover it can be controlled by an android app with aid of bluetooth.
+
+## Motivation
+Our Bot can move efficiently on any plane surface, the almost frictionless design reduces wear and tear and saves energy the versatility of a hovercraft is well exploited by the US NAVY for shipment, which was our motivation, where we tried to further improve the control mechanism.
+
+## Electronics Aspect of the Design 
+1. Electronic Component Used
+     1. BLDC Motors
+     2. Arduino UNO
+     3. Batteries
+     4. Air blower 
+     5. Bluetooth module HC-05
+     6. ESCs
+     7. Android device
+     8. Connecting wires and jumpers
+2. 2. Arduino Uno was used for motion of BLDC motors as well as to connect bluetooth module.
+3. A bluetooth module was used for transmission of commands from android device to our hoverpod.  
+4. All the BLDC motors were powered through Li-Po 11.1V dc battery.
+![circuit](https://github.com/shubhamverma989630/Hoverpod/blob/master/Electronic%20compo.jpg)
+
+
+
+## Mechanical Aspect of the Design
+It stands on a circular wooden board and comprises of 3 BLDC motors and a blower. Below it it consists of an inflated air tube which allows it to levitate over ground.
+
+
+## Cost Structure
+|COMPONENTS | COST(₹)|
+|----|------|
+|Android Device|10000|
+|Blower|750|
+|Arduino Uno|500|
+|ESC motor driver(3)|480|
+|Bluetooth module|350|
+|Voltmeter|160|
+|BLDC motors(3)|1050|
+|Li-Po battery(2)|2200|
+|Other Components|2000|
+|Total|17490|
+
+##  Final Changes and Modification in design
+The structure was inclined due to unhealthy weight balancing so we need to balance it. And also it required some error correction in steering it.
+
+## Working
+Air blower blows air in the tube which consists of small pores at the lower side of it so that it creates an air cushion over which structure can hover. It is controlled by 3 BLDC motors which helps to steer it. And BLDC is controlled through arduino with the aid of bluetooth module. 
+
+ ## Limitations
+ - It can lift only upto 15 kgs.
+ - As friction is not present,it is a bit difficult to steer it.
+ - Blower requires a high power input so it need to be connected through wire.
+ ## Future Improvements
+ - To make it wireless
+ - To improve controlling.
+ - To increase weight lifting capability.
+
+## Team Members
+

```

Comment:
Name of team members should be in alphabetical order

Reply:
Folder Name should be Mechanical Design not Mechanical Aspect. Add jpeg Solidworks design also in Mechanical Design folder.